docs/sphinx/quickstart.rst

```diff
@@ -76,6 +104,30 @@ This indexes a document with the index API:
         },
     )
 
+You can also index multiple documents at once with the bulk API:
+
+.. code-block:: python
+
+    def generate_operations(documents, index_name):
+      operations = []
+      for i, document in enumerate(documents):
+          operations.append({"index": {"_index": index_name, "_id": i}})
+          operations.append(document)
+      return operations
+
+    client.bulk(index=index_name, operations=generate_operations(books, index_name), refresh=True)
```

Comment:
What are your thoughts on *not* documenting this? It does not scale to a large number of documents and is more verbose. The helper is the way to go. I think I would even mention that `client.bulk` should be avoided.

Reply:
Added these changes!